Drop-In/src/test/java/com/braintreepayments/api/DropInActivityUnitTest.java

```diff
@@ -602,6 +602,139 @@ public void onSupportedPaymentMethodSelected_withTypeUnknown_showsAddCardFragmen
         assertNotNull(mActivity.getSupportFragmentManager().findFragmentByTag("ADD_CARD"));
     }
 
+    @Test
+    public void tokenizeCard_onError_whenErrorNotErrorWithResponse_finishesWithError() {
```

Comment:
Does this need an equivalent test for `whenError Is ErrorWithResponse`

Reply:
Good call - the handling of the ErrorWithResponse is tested with `CardDetailsFragment`, but I added a test that the error is set in the `dropInViewModel` with it is `ErrorWithResponse`